BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/FileDiagnosticsTelemetryModule.cs

```diff
@@ -103,7 +99,15 @@ public string LogFilePath
         }
 
         /// <summary>
-        /// Initializes the telemetry module.
+        /// Gets or sets the self-diagnostics configuration string used to setup this module.
+        /// </summary>
+        /// <remarks>
+        /// This module can be setup by the Self-Diagnostics Environment Variable and we want to include that string in the file.
```

Comment:
I'd avoid mentioning about Env variables here - this class contains no means to read from Env variable. That job is done by TCFactory.